src/templates/fr/2019/contributors.html

```diff
@@ -299,9 +294,11 @@ <h1 class="title title-lg">
       <img src="/static/images/avatars/9.jpg" height="100" width="100" loading="lazy" />
   </div>
   <div class="contributors {{ ' '.join(config.teams.keys()) }}">
-    <div class="contributors-count"><span id="filtered-contributors">{{ config.contributors.items() | length }}</span> personnes</div>
     <div class="filter-container">
-      <p>Filtrer par équipe&nbsp;:</p>
+      <p>
+        Filtrer par équipe&nbsp;:
```

Comment:
Any reason for this `&nbsp;` @borisschapira ?

Reply:
Noticed that too, wanted to change but as this got to master, I thought there might be a reason for that.

Reply:
I saw remove. Unless @borisschapira gives good reason to keep it.

Reply:
French microtypography. We have non-break spaces before ":", ";", "!", "?", between numbers and units ("14&nbsp;%")…

Reply:
OK. As long as there's a reason. Resolving this with no change then.